File: codegen/codegen-erlang/src/test/resources/ir/host_label_helpers.expected.erl
build_host(#get_tenant_data_input{tenant = Tenant}, Config) ->
    BaseUrl = maps:get(base_url, Config, <<>>),
    {_Scheme, Authority} = runtime_utils:split_base_url(BaseUrl),
    Prefix = <<(uri_encode(to_binary(Tenant)))/binary, ".">>,
    <<Prefix/binary, Authority/binary>>.

File: codegen/codegen-elixir/src/test/resources/ir/host_label_helpers.expected.ex
defp build_host(%Types.GetTenantDataInput{tenant: tenant}, config) do
  base_url = Map.get(config, :base_url, "")
  {_scheme, authority} = Utils.split_base_url(base_url)
  prefix = URI.encode(Kernel.to_string(tenant)) <> "."
  prefix <> authority
end
